policy/watcher.go

```diff
@@ -1163,14 +1338,39 @@ func (w *watcher) run(args ...string) error {
 }
 
 func (w *watcher) insertBaseRules() error {
+	err := w.deleteBaseRules()
+	err = w.deleteBaseRulesOut()
+	err = w.insertBaseRulesOut()
 	checkRule := fmt.Sprintf("iptables -w -C %v -d %v -s %v -j %v",
 		hookToChain, w.defaultSubnet, w.defaultSubnet, cattleNetworkPolicyChainName)
 	if executeCommandNoStderr(checkRule) != nil {
 		addRule := fmt.Sprintf("iptables -w -I %v -d %v -s %v -j %v",
 			hookToChain, w.defaultSubnet, w.defaultSubnet, cattleNetworkPolicyChainName)
-		return executeCommand(addRule)
+		err = executeCommand(addRule)
 	}
-	return nil
+	return err
+}
+
+func (w *watcher) insertBaseRulesOut() error {
+	var err error
+	checkRule := fmt.Sprintf("iptables -w -C %v ! -d %v -s %v -j %v",
+		hookToChain, w.defaultSubnet, w.defaultSubnet, cattleNetworkPolicyChainNameOut)
+	if executeCommandNoStderr(checkRule) != nil {
+		addRule := fmt.Sprintf("iptables -w -I %v ! -d %v -s %v -j %v",
+			hookToChain, w.defaultSubnet, w.defaultSubnet, cattleNetworkPolicyChainNameOut)
+			err = executeCommand(addRule)
+	}
+	checkRuleDNS := fmt.Sprintf("iptables -w -C %v -d %v -s %v -j %v",
+	cattleNetworkPolicyChainNameOut, "169.254.169.250", w.defaultSubnet, "ACCEPT")
+	if executeCommandNoStderr(checkRuleDNS) != nil {
+		addRule := fmt.Sprintf("iptables -w -I %v 1 -d %v -s %v -j %v",
+		cattleNetworkPolicyChainNameOut, "169.254.169.250", w.defaultSubnet, "ACCEPT")
```

Comment:
Grab the metadata IP from the CLI argument?
https://github.com/rancher/network-policy-manager/blob/master/main.go#L22